prime-router/src/main/kotlin/transport/RESTTransport.kt

```diff
@@ -111,8 +111,11 @@ class RESTTransport(private val httpClient: HttpClient? = null) : ITransport {
                         val httpHeaders = getHeaders(restTransportInfo, reportId)
                         var accessToken: String? = null
 
-                        if (restTransportInfo.authType == "apiKey") {
+                        if (restTransportInfo.authType == "Api Key") {
```

Comment:
We are using apiKey for ELIMS why do we need to change it to Api Key?

Reply:
Using apiKey, Gitleaks doesn't allow me to commit.  I wonder, how did you commit with using "apiKey"?

Reply:
@oslynn You should be able to exclude/ignore that warning in the gitleaks config